external/lfric_infrastructure/preprocessed/apps/flux_direction_mod.f90

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Why is the whole `preprocessed` directory under revision control?

Reply:
See https://github.com/stfc/PSyclone/pull/3116#issuecomment-3235738784